### ISSUE TITLE
Latest firmware

### ISSUE THREAD

**Issue by cwhittin** [johnjore/cflcdpanelchalkelec#1]
For me this seemed to stop working with the LCDs latest firmware.  All the options disappear when I open the plugin,except for the Auto-brightness button (which now has no effect.)  I'm using the 7" HD LCD with dual touch.

**Comment by johnjore** [comment]
Ok. Not much I can do about that. You're probably better off asking the manufacturer to fix the firmware ?

**Comment by cwhittin** [comment]
You want the manufacturer to comply with your Centrafuse plug-in?  :)

**Comment by johnjore** [comment]
Nope, but I suspect you want the plugin to remain working... I didn't upgrade the firmware on my device, so I'm ok... :)

And, I'm not sure you understand that I can't fix firmware issues. If the API's have been removed in latest firmware, there is nothing I can do to bring them back. Only the vendor can do that...

**Comment by cwhittin** [comment]
Oh no I fully understand, I didn't expect the API went away, just some
minor changes maybe that prevent your plugin from working.   The newer
firmware fixed many issues, such as a bad algo for the photo sensor,  bugs
with the backlight, and some features.

Changelog:  https://drive.google.com/open?id=0B7BWcJPjt1LNOXpNU1VBZ2dQV1U

Which is the latest firmware your plugin is compatible with?

Thanks.

**Comment by cwhittin** [comment]
And I'm not trying to be rude, sorry, your plugin is free and highly
appreciated,  I was just hoping it would be a very easy fix.   I will do my
best to look over your source and see if I can figure out what happened by
debugging.   Thanks.
